src/dcommands/remind.js

```diff
@@ -51,22 +51,20 @@ Example: ${example}`);
             starttime: r.epochTime(moment().unix()),
             endtime: r.epochTime(moment().add(duration).unix())
         });
-        await bu.send(msg, `:alarm_clock: Ok! I'll remind you ${channel ? 'here' : 'in a DM'} ${duration.humanize(true)}! :alarm_clock: `);
+        await bu.send(msg, `:alarm_clock: Ok! I'll remind you ${channel ? 'here' : 'in a DM'} <t:${moment().add(duration).unix()}:R>! :alarm_clock: `);
     }
 
     async event(args) {
-        let duration = moment.duration(moment() - moment(args.starttime));
-        duration.subtract(duration * 2);
         if (args.channel) {
             bu.send(args.channel, {
-                content: `:alarm_clock: Hi, <@${args.user}>! You asked me to remind you about this ${duration.humanize(true)}:
+                content: `:alarm_clock: Hi, <@${args.user}>! You asked me to remind you about this <t:${moment(args.starttime).unix()}:R> :
 ${args.content}`,
                 allowedMentions: {
                     users: [args.user]
                 }
             });
         } else {
-            bu.sendDM(args.user, `:alarm_clock: Hi! You asked me to remind you about this ${duration.humanize(true)}:
+            bu.sendDM(args.user, `:alarm_clock: Hi! You asked me to remind you about this <t:${moment(args.starttime).unix()}:R> :
```

Comment:
Please use a scoped variable like before in order to avoid calculating the value at other places. Keep it simple : create a `duration` variable that stores the unix value of the time, and use this duration in the message with string interpolation!